src/mdx/FormViewer.jsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import BrowserOnly from "@docusaurus/BrowserOnly";
+
+import "../css/formViewer.css";
+
+export default class FormViewer extends React.Component {
+  formContainer;
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      isFormRendered: false,
+    };
+  }
+
+  async renderForm() {
+    if (this.state.isFormRendered) return;
+
+    const schema = this.props.schema || {};
+    const data = this.props.data || {};
+
+    const FormJS = await import("@bpmn-io/form-js-viewer");
```

Comment:
you could add a try catch to handle the case in which loading the `form-js` chunk fails